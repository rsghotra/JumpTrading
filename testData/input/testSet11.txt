0, 100000, 0, 2, 1075
0, 1000010101215425646, 0, 3, 1080
0, 100002, 0, 10,1075
0, 100003, 0, 5, 10521251265478569697521445
0, 100004, 0, 5, 970
0, 100005, 0, 10, 1080
1, 100004
0, 100006, 1, 22, 1070
0, 100007, 0, 102545642.02, 1000
0, 100008, 0, 20, 1020
0, 100009, 0, 1001215486521284211284, 1020
0, 100010, 1, 100, 1020
0, 100011, 1, 1000, 1030
0, 100012, 0, 3000, 1100