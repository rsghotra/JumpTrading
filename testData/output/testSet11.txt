ERROR:  Bad Input: Unexpected or out of range datatype detected.
ERROR:  Bad Input: Unexpected or out of range datatype detected.
ERROR:  Bad Input: Unexpected or out of range datatype detected.
2,1,1075
3,100010
4,100000,1
