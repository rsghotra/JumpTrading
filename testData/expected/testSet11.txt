2,1,1075
3,100010
4,100000,1
